docs/docs/t-digest.md

```diff
@@ -62,62 +62,56 @@ You measure the IP packets transferred over your network each second and try to
 
 ## Examples
 
-#### Creating a t-digest
+We'll demonstrate creating a t-digest with an initial compression of 100, adding items to it, and checking the estimated percentile of a value. Note that the `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default is 100. Higher values mean more accuracy. Note also that unlike some of the other probabilistic data structures, the `TDIGEST.ADD` command will not create a new structure if the key does not exist.
```

Comment:
I don't see where you "check the estimated percentile of a value" in the example.

```suggestion
In the following example, you'll create a t-digest with an initial compression of 100, add items to it, and check the estimated percentile of a value. The `COMPRESSION` argument is used to specify the tradeoff between accuracy and memory consumption. The default value is 100. Higher values mean more accuracy. Note: unlike some of the other probabilistic data structures, the `TDIGEST.ADD` command will not create a new structure if the key does not exist.
```

Reply:
I wouldn't use "initial compression" because it suggests that it can be changed or is being changed automatically. I would use just "compression" instead. 
I also suggest changing "check the estimated percentile" to "query ...".